### PR TITLE
Created Swedish translation

### PR DESCRIPTION
**Pull request by cederberget**
Created translation and stuff

### CODE REVIEW

File: js/strings.js [hunk 1]
```diff
@@ -1,10 +1,101 @@
 const stringTables = {
-    se: {
-        header_services: 'stuff',
-        hero_tagline: 'Swedish tagline'
+    en: {
```
Comment (YannikH) [inline js/strings.js:2]: I'd prefer we don't put english in the string tables, so the HTML remains the source of truth for English.

**Review by YannikH — COMMENTED** [review]
I suggest we remove the English text from the stringtable json to avoid confusion about what's the source of truth for English.

File: js/strings.js (hunk 1 above)
Comment (cederberget) [reply js/strings.js:2]: The problem is it's the only legend we have for the strings that exist to be translated.

**Review by YannikH — APPROVED** [review]
LGTM

**Pull request merged**